.eslintignore

```diff
@@ -0,0 +1,2 @@
+src/**/*.test.tsx
+**/*.d.tsx
```

Comment:
```suggestion
**/*.d.ts
**/node_modules/**/*
src/**/*.test.tsx
```

Ambient-type declaration files don't contain JSX. So they shouldn't the `.tsx` suffix. Also, you'll want to add an entry for `node_modules`.